aide_design/constants.py

```diff
@@ -6,6 +6,9 @@
 
 Last modified: Fri Jul 7 2017
```

Comment:
Take off this Sage business and last modified stuff... it's all stored on GitHub.